src/backend/sev/snp/firmware/linux.rs

```diff
@@ -0,0 +1,99 @@
+// SPDX-License-Identifier: Apache-2.0
+
+//! Operations for managing the SEV platform.
+
+use super::super::{Error, Indeterminate, Version};
+use super::{Build, SnpPlatformStatus, State, Status, TcbStatus};
+
+use iocuddle::{Group, Ioctl, WriteRead};
+
+use std::fs::{File, OpenOptions};
+use std::marker::PhantomData;
+use std::os::unix::io::{AsRawFd, RawFd};
+
+// These enum ordinal values are defined in the Linux kernel
+// source code: include/uapi/linux/psp-sev.h
+impl_const_id! {
+    pub Id => u32;
+    // […]
```

Comment:
We will also need `GetId = 8` here for #1020 , but I can move that myself from SEV crate in the PR